### PR TITLE
Added portable lasers and more kinetic multi-tool upgrades.

### PR DESCRIPTION
**Pull request by iTitus**

### CODE REVIEW

File: src/main/java/buildcraftAdditions/reference/Variables.java [hunk 1]
```diff
@@ -32,6 +32,7 @@
 		@Deprecated
 		public static final int KINETIC_TOOL = 72;
 		public static final int KINETIC_MULTI_TOOL = 78;
+		public static final int PORTABLE_LASER = 79;
```
Comment (AEnterprise) [inline src/main/java/buildcraftAdditions/reference/Variables.java]: 78 and 79 come after 77 and not 72 IIRC

**Comment by iTitus** [comment]
Please review this. I did some changes to the ItemKineticMultiTool that I need in my other PR...
Sorry for that :P

**Comment by iTitus** [comment]
Finished D

**Comment by AEnterprise** [comment]
looks good

**Pull request merged**